config/sudoers

```diff
@@ -35,6 +35,7 @@ vx-admin ALL=(root:ALL) NOPASSWD: /usr/sbin/reboot
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/create-cert
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/auth/src/intermediate-scripts/sign-message
 vx-services ALL=(root:ALL) NOPASSWD: /vx/code/vxsuite/libs/usb-drive/scripts/*
+vx-services ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/hash-signature.sh
```

Comment:
I think we'll want to update `config/sudoers-for-dev`, too

Reply:
Good catch, almost always forget about this file.